app/src/main/java/tech/ula/ui/AppListFragment.kt

```diff
@@ -158,4 +174,26 @@ class AppListFragment : Fragment() {
     private fun passDataToActivity(data: String) {
         dataPasser.onFragmentDataPassed(data)
     }
-}
+
+    private fun select_client_preference(selectedApp: App) {
```

Comment:
Stick with camel case for kotlin side code. Snake case is used to denote xml entities.